index.html

```diff
@@ -0,0 +1,107 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,400;0,500;0,700;1,400&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+        integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div class="container-header">
+            <h1 class="logo-title">Marlyn Mayienga</h1>
+            <nav class="nav-bar">
+                <ul>
+                    <li><a href="#"></a>Portfolio</a></li>
+                    <li><a href="#"></a>About</a></li>
+                    <li><a href="#"></a>Contact</li>
+                    <li><a href="#"><i class="far fa-envelope"></i></a></li>
+                </ul>
+            </nav>
+            <div class="hamburger-btn">
+                <i class="fas fa-bars"></i>
+            </div>            
+        </div>
+    </header>
+
+    <main class="content">
+        <h1 class="name">Marlyn Mayienga</h1>
+        <img class="first-img" src="../My_Portfolio/circle 2 header.svg" alt="circle 2 header">
+        <h1>Software Developer</h1>
+        <p class="first-paragraph">I can help you build a product, feature or website<br> Look through some of my work
+            and experience!If <br>
+            you like what you see and have a project you need<br> coded, don’t hestiate to contact me.</p>
+
+        <hr>
+        <!-- TECHNLOGIES -->
+        <div class="technologies">
+            <div class="technologies-item">
+                <div class="technologies-title">Languages</div>
+                <ul>
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
+            <div>
+                <div class="technologies-item">
+                    <div class="technologies-title">Frameworks</div>
+                    <ul>
+                        <li>BootStrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RPSec</li>
+                        <li>CapyBase</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+            </div>
+            <div>
+                <div class="technologies-item">
+                    <div class="technologies-title">Skills</div>
+                    <ul>
+                        <li>Codekit</li>
+                        <li>GitHub</li>
+                        <li>CodePEn</li>
+                        <li>GitLab</li>
+                        <li>Terminal</li>
+                    </ul>
```

Comment:
- [ ] If you see the Figma design then you see that this content should be in the next line. so kindly fix it.
- [ ] Kindly make the `work` para just after the social icons and remove the one because there are two `work ` para in left and right corner
-  Your design
![his design](https://user-images.githubusercontent.com/81727988/164343845-5412dc4d-2ef1-45d6-b1ab-0fa5c6111e5e.png)
- Figma
![figma design](https://user-images.githubusercontent.com/81727988/164343867-141cbe87-90e4-486c-b401-db28169e3ef5.png)
a design